templates/ez_segmenter.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>This should definitely be cleared.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>I think this was a keyboard interrupt - should be fine?</p>

Reply:
<p>If this gets committed to main the user will see the full KeyboardInterrupt stack trace, which might be confusing.</p>

Reply:
<p>I see what you mean - cleared.</p>